CompanyEmployees.Infrastructure.Persistence/Configurations/AttendanceConfiguration.cs

```diff
@@ -0,0 +1,28 @@
+using CompanyEmployees.Core.Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace CompanyEmployees.Infrastructure.Persistence.Configurations;
+
+public class AttendanceConfiguration : IEntityTypeConfiguration<Attendance>
+{
+    public void Configure(EntityTypeBuilder<Attendance> builder)
+    {
+        builder.HasKey(a => a.Id);
+
+        builder.HasOne(a => a.Employee)
```

Comment:
Consider specifying a navigation property for the inverse relationship in Employee, such as a collection of Attendances. This improves EF's understanding of the relationship and helps with more efficient queries and modeling.

**Suggested change**
```suggestion
// In the Employee entity: public ICollection<Attendance> Attendances { get; set; } = new List<Attendance>();
```